.run_cheri_qemu_and_test.py

```diff
@@ -26,7 +26,8 @@ def run_tests(qemu: boot_cheribsd.QemuCheriBSDInstance, args: argparse.Namespace
 
     # Run command on host to test the executed client
     os.chdir(f"{args.build_dir}/build")
-    subprocess.run(["/usr/bin/ctest", "-V", "--output-on-failure"], check=True)
+    subprocess.run(["../tests/prep_tests.sh"], check=True)
+    subprocess.run(["/usr/bin/ctest", "--output-on-failure"], check=True)
```

Comment:
Should we not use whatever `ctest` is in `$PATH` rather than hardcoding  `/usr/bin`  .  There might be other distributions or OS'es, where the binary is placed in other locations

Reply:
Fixed in https://github.com/capablevms/CHERI-ELF-comp/pull/5/commits/9bffb24a00520d8ddf1966dd3a79b84a1298ff8d.